package.json

```diff
@@ -19,7 +19,8 @@
     "watch": "jest --watch --no-coverage",
     "playroom:start": "playroom start",
     "playroom:build": "playroom build",
-    "build": "yarn playroom:build"
+    "build": "yarn playroom:build",
+    "release": "yarn dist && changeset publish"
```

Comment:
qq - do we still need the `prepublishOnly` script on line 15? Wondering if we will end up calling `yarn run dist` twice: once when `release` is called, and then again when changeset tries to publish (because the prepublishOnly script might be called automatically)?

Reply:
That's a good point. I don't think we need the `prepublishOnly` script anymore. I'll remove it.